docs/plans/FFE_OPENCLAW_AUTONOMOUS_DERRR_V1.md

```
@@ -0,0 +1,447 @@
+then OpenClaw should stop autonomous advancement and hand off to the human.
+
+---
+
+## DERRR mapping in V1
+
+V1 should not force the watcher to emit full DERRR phases directly.
+Instead, watcher targets should map into DERRR behavior.
+
+### Watcher target -> DERRR behavior
+
+#### `document`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)